fury/actors/tests/test_peak.py

```diff
@@ -33,7 +33,6 @@ def generate_peaks():
     peaks_vals[0, 1, 2, :] = np.array([0, .7, 0, 0, 0])
     return peaks_dirs, peaks_vals, np.eye(4)
 
-
```

Comment:
Removing this line is rising the following PEP8 issue. `PEP 8: E302 expected 2 blank lines, found 1`